### PR TITLE
MT-5411: Running ecs task rely on task exit code

### PR DESCRIPTION
**Pull request by IgorDobryn**
### Motivation
Prevent github action execution if ecs task exit with unsuccessful status code

### Changes
1. Extract `runEcsTask`
2. Exit from `runEcsTask` if command finishes with unsuccessful status code
3. Improve test coverage

### CODE REVIEW

**Review by VladimirTaytor — APPROVED** [review]

**Review by charlesjom — APPROVED** [review]

File: run-ecs-task/__tests__/runEcsTask.test.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                              lastStatus: 'STOPPED',
+                              containers: [{
+                                name: 'EcsContainerDefinition',
+                                exitCode: 0
+                              }]
+                            }]
+                          })
+
+                        }) => ({
+    describeServices,
```
Comment (r4do) [inline run-ecs-task/__tests__/runEcsTask.test.js:51]: we need to add prettier here 🤔

File: run-ecs-task/__tests__/runEcsTask.test.js [hunk 2]
```diff
@@ -0,0 +1,253 @@
+const runEcsTask = require('../runEcsTask')
+
+describe('runEcsTask', () => {
+  const buildMock = (data) => (
+    jest.fn().mockReturnValue({
+      promise: () => Promise.resolve(data)
+    })
+  )
+  const buildEcsMock = ({
+                          describeServices = buildMock({
```
Comment (r4do) [inline run-ecs-task/__tests__/runEcsTask.test.js:10]: would be great to update the formatting, as it's not consistent across the file.

**Review by r4do — APPROVED** [review]

File: run-ecs-task/__tests__/runEcsTask.test.js (hunk 2 above)
Comment (IgorDobryn) [reply run-ecs-task/__tests__/runEcsTask.test.js:10]: I guess it was autoformatting with some default settings from my IDE. I'd prefer to leave this as is until we add prettier. Prettier should solve this much better and everywhere

**Review by leonid-shevtsov — COMMENTED** [review]
:+1:

**Pull request merged**